tox.ini

```diff
@@ -13,7 +13,7 @@ show-source = True
 [flake8-rst]
 filename = *.rst *.py
 max-line-length = 120
-ignore = F821  ; TODO remove me when all examples in docstrings will be executable
+ignore = F821, F401  ; TODO remove me when all examples in docstrings will be executable
```

Comment:
Is the intent of adding `F401` here to ignore *all* unused imports *everywhere*? (That seems a big change from the past.)

Reply:
This should apply only to docstrings (flake8-rst), not "actual" code.

It's actually working around a bug where one import in fasttext.py was actually used but still triggered this error, mysteriously.

Let me add a better comment here (and maybe report that bug upstream).

Reply:
I went down the rabbit hole of debugging this. flake8-rst (added in PR #2192) is quite flaky, doesn't even work with up-to-date versions of flake8: https://gitlab.com/pycqa/flake8/-/issues/641

I'm inclined to remove flake8-rst altogether, rather than waste more time on this.